hack/dockerized

```diff
@@ -12,7 +12,8 @@ fi
 
 fail_if_cri_bin_missing
 
-KUBEVIRT_BUILDER_IMAGE="quay.io/kubevirt/builder:2109081533-c2a4c43a6"
+#This is the builder from release-0.50
+KUBEVIRT_BUILDER_IMAGE="quay.io/kubevirt/builder:2112231627-152540e23"
```

Comment:
This is the builder for release-0.50 which include golang 1.17. Is it okay to refer to this builder without updating Dockerfile and doing a new builder build?

cc @brianmcarey @xpivarc